src/commands/mobile_app/mod.rs

```diff
@@ -0,0 +1,48 @@
+use anyhow::Result;
+use clap::{ArgMatches, Command};
+
+use crate::utils::args::ArgExt;
+
+pub mod upload;
+
+macro_rules! each_subcommand {
+    ($mac:ident) => {
+        $mac!(upload);
+    };
+}
+
+pub fn make_command(mut command: Command) -> Command {
+    macro_rules! add_subcommand {
+        ($name:ident) => {{
+            command = command.subcommand(crate::commands::mobile_app::$name::make_command(
+                Command::new(stringify!($name).replace('_', "-")),
+            ));
+        }};
+    }
+
+    command = command
+        .about("(Experimental) Manage mobile apps.")
```

Comment:
[nit] this is the style we typically use for marking commands. We don't have any other experimental commands at the moment, but this style aligns with how we mark commands as deprecated using `[DEPRECATED]`

```suggestion
        .about("[EXPERIMENTAL] Manage mobile apps.")
```